src/myPage/MyPage.jsx

```diff
@@ -93,6 +98,25 @@ export class MyPage extends React.Component {
     }
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Should be possible to do this without using `getDerivedStateFromProps`. Maybe just remove the search query on modal close?

https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html